dv/management/commands/import_news.py

```diff
@@ -0,0 +1,46 @@
+import json
+
+from urllib.request import urlopen
+from datetime import datetime
+from pytz import timezone
+
+from django.core.management.base import BaseCommand
+from dv.models import News
+
+ENDPOINT = 'http://eeagrants.org/rest/articles?page={}'
+
+
+class Command(BaseCommand):
+    help = 'Import the file given as argument'
+
+    def handle(self, *args, **options):
+        page = 0
+        data = []
+        News.objects.all().delete()
+        while page == 0 or data:
+            with urlopen(ENDPOINT.format(page)) as url:
+                data = json.loads(url.read().decode())['posts']
+                print('Importing {} news from page {}'.format(len(data), page))
+                for item in data:
+                    self._save(item)
+                page += 1
+
+    def _save(self, item):
+        try:
+            self.stdout.write('Importing post %s' % item['link'])
+            news, created = News.objects.get_or_create(link=item['link'])
```

Comment:
... and if not created, and no change in updated, raise something here that stops everything